telegram/ext/commandhandler.py

```diff
@@ -159,3 +169,147 @@ def collect_optional_args(self, dispatcher, update=None, check_result=None):
 
     def collect_additional_context(self, context, update, dispatcher, check_result):
         context.args = check_result
+
+
+class PrefixHandler(CommandHandler):
+    """Handler class to handle custom prefix commands
+
+    This is a intermediate handler between :class:`MessageHandler` and :class:`CommandHandler`.
+    It supports configurable commands with the same options as commandhandler. It will respond to
+    every combination of :attr:`prefix` and :attr:`command`.
+
+    Examples::
+
+        Single prefix and command:
+
+            PrefixHandler('!', 'test', callback) will respond to '!test'.
+
+        Multiple prefixes, single command:
+
+            PrefixHandler(['!', '#'], 'test', callback) will respond to '!test' and
+            '#test'.
+
+        Miltiple prefixes and commands:
+
+            PrefixHandler(['!', '#'], ['test', 'help`], callback) will respond to '!test',
+            '#test', '!help' and '#help'.
+
+    Attributes:
+        prefix (:obj:`str` | List[:obj:`str`]): The prefix(es) that will precede :attr:`command`.
+        command (:obj:`str` | List[:obj:`str`]): The command or list of commands this handler
```

Comment:
Missing a full stop (.)

Reply:
? On the next line at the end of the sentence...

Reply:
Oh derp, sorry :P